ui/app/controllers/allocations/allocation/task/index.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  network: computed.alias('model.resources.networks.firstObject'),
+  ports: computed('network.reservedPorts.[]', 'network.dynamicPorts.[]', function() {
+    const ports = this.get('network.reservedPorts')
```

Comment:
can probably clean this up a bit with `computed('network.reservedPorts.[]', 'network.dynamicPorts.[]', () => this.get('network.reservedPorts') ...`